poutyne/framework/callbacks/gradient_tracking.py

```diff
@@ -0,0 +1,84 @@
+import torch
+from torch import tensor
```

Comment:
Use `torch.Tensor` instead of `tensor`.

Reply:
de472b1

Reply:
I meant using `torch.tensor` in the code instead of importing `tensor` from `torch`.

Reply:
Anyway, I will test the diffence running time between using only tensor vs NumPy array.

Reply:
Changed to NumPy d9a6757